common-compose.yml

```diff
@@ -1,35 +1,39 @@
 # MySQL designed for container-native deployment on Joyent's Triton platform.
```

Comment:
FYI, I've started work on fixing up this repo in https://github.com/autopilotpattern/mysql/pull/14